### PR TITLE
Lra beta

### PR DESCRIPTION
**Pull request by idlewis**
Update flags on the LRA related features from noship to beta

### CODE REVIEW

**Comment by idlewis** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpLRA-1.0/io.openliberty.mpLRA-1.0.feature [hunk 1]
```diff
@@ -14,5 +14,5 @@ IBM-API-Package: \
 -bundles=io.openliberty.microprofile.lra.1.0.internal, \
          io.openliberty.org.jboss.narayana.rts, \
          com.ibm.ws.org.jboss.logging
-kind=noship
-edition=full
+kind=beta
+edition=core
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpLRA-1.0/io.openliberty.mpLRA-1.0.feature:18]: Need to add `WLP-Activation-Type: parallel`

**Comment by idlewis** [comment]
#build

**Comment by idlewis** [comment]
@tevans78 I've added the parallel activation as requested. Could you take another look at the review?

**Review by tevans78 — APPROVED** [review]

**Comment by idlewis** [comment]
#build

**Comment by idlewis** [comment]
#run-libby-bot

**Pull request merged**